editor/editor_help.cpp

```diff
@@ -308,7 +308,7 @@ void EditorHelp::_add_type(const String &p_type, const String &p_enum, bool p_is
 	bool can_ref = !p_type.contains("*") || is_enum_type;
 
 	String link_t = p_type; // For links in metadata
-	String display_t = link_t; // For display purposes
+	String display_t; // For display purposes
```

Comment:
```suggestion
	String display_t; // For display purposes.
```
Just enforcing the style when changing the line here

Reply:
That makes sense, thanks!